<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
  background-color: #eff4ff;
}
</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLineEdit" name="password_lineedit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
  color: #1d1d1d;
  background-color: #ecf0f1;
  border: 2px solid #2c3e50;
  border-radius: 8px;
  padding-left: 25px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  text-align: left;
  padding-right: 50px;
  background-image: url(:/qss/icon/password.png);
  background-repeat:no-repeat;
  background-position: center left;
}

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="user_type_groupbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
  border: 1px solid #1d1d1d;
  border-top: 1px solid #1d1d1d;
  padding: 5px
}

 
QGroupBox::title
{
   
border: 1px solid #1d1d1d;
    border-bottom: none;
    border-letf:none;
    border-right:none;
     padding: -20 5px;  
    
}</string>
   </property>
   <property name="title">
    <string>user type</string>
   </property>
   <widget class="QRadioButton" name="customer_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
    spacing: 5px;
}
 
QRadioButton::indicator{
    width: 15px;
    height: 15px;
}
 
QRadioButton::indicator:checked{
    image:url(:/qss/icon/radio-on-button.png);
}

QRadioButton::indicator:unchecked{
    image:url(:/qss/icon/radio-off.png);
}</string>
    </property>
    <property name="text">
     <string>customer</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="hotelmanager_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
    spacing: 5px;
}
 
QRadioButton::indicator{
    width: 15px;
    height: 15px;
}
 
QRadioButton::indicator:checked{
    image:url(:/qss/icon/radio-on-button.png);
}

QRadioButton::indicator:unchecked{
    image:url(:/qss/icon/radio-off.png);
}</string>
    </property>
    <property name="text">
     <string>hotel manager</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="administrator_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton{
    spacing: 5px;
}
 
QRadioButton::indicator{
    width: 15px;
    height: 15px;
}
 
QRadioButton::indicator:checked{
    image:url(:/qss/icon/radio-on-button.png);
}

QRadioButton::indicator:unchecked{
    image:url(:/qss/icon/radio-off.png);
}</string>
    </property>
    <property name="text">
     <string>administrator</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>247</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: #1d1d1d;
  background-color: #bdc3c7;
  border: 1px solid #2c3e50;
  border-radius: 8px;
  padding: 3px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  background-repeat:no-repeat;
  background-position: center left;
}

QPushButton:hover, QPushButton:pressed , QPushButton:checked
{
  background-color:  #95a5a6;
}


</string>
   </property>
   <property name="text">
    <string>Register Now</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Login_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft New Tai Lue</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  color: #ffffff;
  background-color: #2d89ef;
	border:none;
  padding: 3px;
  font-family:&quot;Microsoft YaHei UI&quot;;
	text-align: center;
}
QPushButton:hover, QPushButton:pressed , QPushButton:checked
{
  background-color:  #3c5bb9;
	text-align: right;
	padding-right: 20px;

}
QPushButton:hover
{
  background-image: url(:/qss/icon/appbar.navigate.next.png);
	background-repeat:no-repeat;
	background-position: center left;
}
QPushButton:pressed, QPushButton:checked
{
  background-image: url(:/qss/icon/appbar.navigate.next.png);
	background-repeat:no-repeat;
	background-position: center left;
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="register_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
    </font>
   </property>
   <property name="text">
    <string>Not a Btrip memeber?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineedit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>69</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
  color: #1d1d1d;
  background-color: #ecf0f1;
  border: 2px solid #2c3e50;
  border-radius: 8px;
  padding-left: 25px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  text-align: left;
  padding-right: 50px;
  background-image: url(:/qss/icon/account.png);
  background-repeat:no-repeat;
  background-position: center left;
}


</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
  color: #1d1d1d;
  background-color: none;
  border: none;
  padding-left: 30px;
  font-family:&quot;Microsoft YaHei UI&quot;;
  text-align: left;
  background-image: url(:/qss/icon/error.png);
  background-repeat:no-repeat;
  background-position: center left;
}</string>
   </property>
   <property name="text">
    <string>error</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
